pkg/convert/json_ng.go

```diff
@@ -0,0 +1,62 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func FromNgJson(b []byte) (model.Messages, error) {
+	incoming := make(map[string]interface{})
+
+	err := json.Unmarshal(b, &incoming)
+	if err != nil {
+		return model.Messages{}, fmt.Errorf("decode NG JSON to go map: %w", err)
+	}
+
+	tag, err := language.Parse(fmt.Sprint(incoming["locale"]))
+	if err != nil {
+		return model.Messages{}, fmt.Errorf("parse NG JSON key: locale : %w", err)
+	}
+
+	msg := model.Messages{
+		Language: tag,
+		Messages: []model.Message{},
+	}
+
+	f := func(in interface{}) {
+		switch v := in.(type) {
+		case map[string]interface{}:
+			for key, value := range v {
+				msg.Messages = append(msg.Messages, model.Message{ID: key, Message: fmt.Sprint(value)})
```

Comment:
no type checking for this one. Could lead to wrong message value. 
e.g. if value is somehow not a string, but json, int or whatever, then it gets parsed as string and added to message field, which is not that we want to achieve.